### PR TITLE
Be able to use make to upgrade the version 1.6

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl [hunk 1]
```diff
@@ -804,8 +804,10 @@ $(INSTANCE_ID_MARKER):
 
 .PHONY: upgrade
 upgrade: .build/requirements.timestamp project.yaml
-	.build/venv/bin/pip install --index-url http://pypi.camptocamp.net/pypi \
-		--find-links https://camptocamp.github.io/c2cgeoportal/index/c2cgeoportal \
+	.build/venv/bin/pip install \
+		--trusted-host pypi.camptocamp.net \
+		--index-url http://pypi.camptocamp.net/pypi \
+		--find-links http://pypi.camptocamp.net/internal-pypi/index/c2cgeoportal \
```
Comment (fredj) [inline c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl]: changed from https to http ?

Comment (sbrunner) [inline c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl]: It's a completely new URL, and pypi.camptocamp.net don't works on https ...

Comment (fredj) [inline c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl]: ok, I didn't noticed that :-)

**Comment by fredj** [comment]
+1

**Pull request merged**